### PR TITLE
Bug/order cancellation policy

### PR DESCRIPTION
**Pull request by Alvtag**
**Description**

**Issues that should be CLOSED by merge of this PR:**
https://guestlogix.atlassian.net/browse/TPP-1034

### CODE REVIEW

File: traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/AnyProductMappingFactory.java [hunk 1]
```diff
@@ -36,6 +37,9 @@ public Product instantiate(JsonReader reader) throws Exception {
                 case "purchaseStrategy":
                     purchaseStrategy = reader.nextString();
                     break;
+                case "cancellationPolicy":
```
Comment (Alvtag) [inline traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/AnyProductMappingFactory.java]: @benRmc @OmarPadierna I don't believe iOS SDK is reading cancellationPolicy, can you guys add?

**Comment by jack-mo** [comment]
You have conflicts

**Comment by Alvtag** [comment]
> You have conflicts

fixed, updated

File: traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/AnyProductMappingFactory.java (hunk 1 above)
Comment (ata-n) [reply traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/AnyProductMappingFactory.java]: @benRmc Can you look into this?

**Pull request merged**